node-check/itb-prepare-hook-lib

```diff
@@ -0,0 +1,579 @@
+#!/bin/bash
+#
+# OSPool / GlideinWMS helper functions.
+#
+# Used for the prepare job hook, particularly to select an appropriate Singularity image
+#
+GWMS_THIS_SCRIPT="$0"
+GWMS_THIS_SCRIPT_DIR=$(readlink -f $(dirname "$0")/..)
```

Comment:
```suggestion
GWMS_THIS_SCRIPT_DIR=$(readlink -f "$(dirname "$0")/..")
```
The result of a `$()` also gets expanded, so if you're going to quote $0 then you should quote $(dirname $0) too.